tests/NSubstitute.Analyzers.Tests.CSharp/DiagnosticAnalyzerTests/NonVirtualSetupAnalyzerTests/ReturnsForAnyArgsAsExtensionMethodTests.cs

```diff
@@ -838,7 +838,7 @@ public void Test()
     }
 }";
 
-            var expectedDiagnostic = new[]
+             var expectedDiagnostic = new[]
```

Comment:
Are these spacing changes fixing alignment? It looks like it makes it out of alignment with the braces but can't tell if it's just the GH diff viewer. Not important at all, just saw this in a number of places in these diffs. Might be worth doing a "Format Document" on these files next time they're changed.

Reply:
Yes, just fixing alignment. Once I've upgraded to new StyleCops.Analyzers(I had to do it as old ones were not able to pick up correctly some of the C# 7.x syntaxes), they start to complain about wrong spacing